### PR TITLE
Issue/2253 resource action log via cli

### LINKED ISSUE #2253: Add support to request the resource action log for a specific resource via the CLI

### PR DESCRIPTION
**Pull request by arnaudsjs**
# Description

 Added support to query the action log of a resource via the CLI.

closes #2253 

# Self Check:

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Type annotations are present
- [x] Code is clear and sufficiently documented
- [x] No (preventable) type errors (check using make mypy or make mypy-diff)
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [x] End user documentation is included or an issue is created for end-user documentation

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: src/inmanta/main.py [hunk 1]
```diff
@@ -801,6 +801,65 @@ def bootstrap_token(client: Client) -> None:
...
+@click.option(
+    "--rvid", help="The resource version ID of the resource", callback=validate_resource_version_id, required=True,
+)
+@click.option("--action", help="Only list this resource action", type=click.Choice([ra.value for ra in ResourceAction]))
+@click.pass_obj
+def resource_action_log_list(client: Client, environment: str, rvid: ResourceVersionIdStr, action: Optional[str]) -> None:
+    tid = client.to_environment_id(environment)
+    """
+        List the resource action log for a specific Resource.
+    """
```
Comment (sanderr) [inline src/inmanta/main.py:829]: This looks like the docstring for `resource_action_log_list`. Should it be one line up?

File: src/inmanta/resources.py [hunk 2]
```diff
@@ -476,10 +476,15 @@ def get_managed(exporter: "export.Exporter", obj: "ManagedResource") -> bool:
     r"(?P<attr>[^=]+)=(?P<value>[^\]]+)\])(,v=(?P<version>[0-9]+))?$"
 )
 
+PARSE_RVID_REGEX = re.compile(
+    r"^(?P<id>(?P<type>(?P<ns>[\w-]+::)+(?P<class>[\w-]+))\[(?P<hostname>[^,]+),"
+    r"(?P<attr>[^=]+)=(?P<value>[^\]]+)\]),v=(?P<version>[0-9]+)$"
```
Comment (sanderr) [inline src/inmanta/resources.py:481]: This allows only for a single attribute, is this always the case?

File: src/inmanta/resources.py [hunk 3]
```diff
@@ -476,10 +476,15 @@ def get_managed(exporter: "export.Exporter", obj: "ManagedResource") -> bool:
     r"(?P<attr>[^=]+)=(?P<value>[^\]]+)\])(,v=(?P<version>[0-9]+))?$"
 )
 
+PARSE_RVID_REGEX = re.compile(
+    r"^(?P<id>(?P<type>(?P<ns>[\w-]+::)+(?P<class>[\w-]+))\[(?P<hostname>[^,]+),"
```
[the remark this answers is not in the document]
Comment (arnaudsjs) [reply src/inmanta/resources.py]: You're right. Sub-namespaces were not captured.

File: src/inmanta/resources.py (hunk 2 above)
Comment (arnaudsjs) [reply src/inmanta/resources.py:481]: Yes. This is always the case.

File: src/inmanta/main.py (hunk 1 above)
Comment (arnaudsjs) [reply src/inmanta/main.py:828]: Indeed. This should be the docstring.

**Pull request merged**